Tests/Expected/Storyboards-iOS/swift2-context-all-ignore-module.swift

```diff
@@ -109,7 +109,7 @@ enum StoryboardSegue {
     case CustomBack
     case Embed
     case NonCustom
-    case Show_NavCtrl = "Show-NavCtrl"
+    case ShowNavCtrl = "Show-NavCtrl"
```

Comment:
TBH, these are all swift 2 changes, which we don't support. I wouldn't bump the version for it.